attestation-agent/attester/src/bin/evidence_getter.rs

```diff
@@ -24,12 +25,16 @@ enum Cli {
 }
 
 #[tokio::main]
-async fn main() {
+async fn main() -> anyhow::Result<()> {
```

Comment:
This is a simple binary, so I think it is fine to panic inside

Reply:
@Xynnn007 one reason to not panic was to make it easier to figure out what's wrong for the user, instead of seeing a stack trace.